src/Feldspar/Onnx/onnxToFeld.hs

```diff
@@ -29,67 +29,111 @@
 
 module Main where
 
-import Onnx.ModelProto
-import Onnx.GraphProto as G
-import Onnx.NodeProto as N
-import Onnx.ValueInfoProto as V
-import Onnx.AttributeProto as A
-import Onnx.TypeProto as T
-import Onnx.TypeProto.Value as TV
-import Onnx.TypeProto.Tensor as TT
-import Onnx.TypeProto.Sequence as TS
-import Onnx.TypeProto.Map as TM
-import Onnx.TensorProto as TP
-import Onnx.TensorProto.DataType as TD
-import Onnx.TensorShapeProto as TSP
-import Onnx.TensorShapeProto.Dimension as TSP
-import Onnx.TensorShapeProto.Dimension.Value as TSP
-
-import Text.ProtocolBuffers
+import qualified Onnx.ModelProto as O
```

Comment:
The trick with qualified imports to avoid the flood of shadowing warnings for single character variables is clever. Sort these lines as well while you're touching them and end up in git blame.

Reply:
Ok.